unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -2629,6 +2629,8 @@ public virtual MetadataWrapper generateMetadataWrapper() {
 
             metaMessage.cameraPosition = m_Camera.transform.position;
             metaMessage.cameraRotation = m_Camera.transform.eulerAngles;
```

Comment:
I'd add the `//to be deprecated` comment to these lines as well, for consistency

Reply:
updated in ef3bfb20c08ec9a7a991b23d7e9008af3c72d132